File: E-Booking/E-Booking/Scripts/KnockoutModels/AdditionalPatientInfoModel.js
function AdditionalPatientInfoModel() {

    var self = this;

    self.AdditionalPatientInfo = ko.observable('').extend({ required: true });
    self.IsInfectious = ko.observable(false);
    self.IsTravellingWithOwnOxygen = ko.observable(false);
    self.IsEscortTravelling = ko.observable(false);
    self.EscortTypeId = ko.observable();
    self.EscortNumberId = ko.observable();
    self.IsBariatric = ko.observable(false);
    self.IsFullLegPlasterPOP = ko.observable(false);
    self.IsElectricWheelchair = ko.observable(false);
    self.IsWheelchairAndLegExtension = ko.observable(false);
    self.IsWaterlow = ko.observable(false);
    self.IsDNACPR = ko.observable(false);
    self.IsDiabetic = ko.observable(false);
    self.IsNuclearMedicineRadioActiveRisk = ko.observable(false);
    self.IsNoneOfAbove = ko.observable(false);

    self.errors = ko.validation.group(self);
}

File: E-Booking/E-Booking.Models/AdditionalPatientInfoModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.ComponentModel.DataAnnotations;

namespace EBooking.Models
{
    public class AdditionalPatientInfoModel
    {
        [Required(ErrorMessage = "Additional Patient Info is required.")]
        public string AdditionalPatientInfo { get; set; }

        public bool IsInfectious { get; set; }

        public bool IsTravellingWithOwnOxygen { get; set; }

        public bool IsEscortTravelling { get; set; }

        public int EscortTypeId { get; set; }

        public int EscortNumberId { get; set; }

        public bool IsBariatric { get; set; }

        public bool IsFullLegPlasterPOP { get; set; }

        public bool IsElectricWheelchair { get; set; }

        public bool IsWheelchairAndLegExtension { get; set; }

        public bool IsWaterLow { get; set; }

        public bool IsDNACPR { get; set; }

        public bool IsDiabetic { get; set; }

        public bool IsNuclearMedicineRadioActiveRisk { get; set; }

        public bool IsNoneOfAbove { get; set; }
    }
}